R/abi.R

```diff
@@ -45,7 +45,7 @@ renv_abi_check <- function(packages = NULL,
   reasons <- unique(tbl$reason)
   if ("Rcpp_precious_list" %in% reasons) {
     packages <- sort(unique(tbl$package[tbl$reason == "Rcpp_precious_list"]))
-    renv_pretty_print(
+    notice(
```

Comment:
Unrelated: I was experimenting with `renv_abi_check` last week and did not see it reporting issues (Linux container using R-4.3) when using mismatched Rcpp & httpuv. I don't have that environment available at the moment, unfortunately. 🚴 ➕ 🌧️ ➕ 💻 ➡️ 💥 😭